controllers/CMakeLists.txt

```
@@ -1,2 +1,2 @@
 add_subdirectory(nugus_controller)
-
+add_subdirectory(teleport_controller)
```

Add newline at end of file to appease the gods
```suggestion
add_subdirectory(teleport_controller)

```